### PR TITLE
botCommands: Regex commands removal

### PR DESCRIPTION
**Pull request by ManonLef**
## Because

We are moving to mostly slash based commands. As discussed in #449 there will be 6 commands, often used in a sentence, that we want to keep text-based. 

EDIT: hit enter prematurely, I'm still editing this template

## This PR

Removes the following text based commands and their test files and snapshots. Note that most of the below commands are still available as slash commands
- bootcamp
- callbacks
- carlosQuote
- debug
- faq
- freelance
- help
- mv
- notes
- shrug (not available as slash)
- time
- windows

These are the commands and services we will keep in the `old-era` botCommands section:
Screenshot 2023-10-16 at 13 23 35

## Issue

Related to #449 

## Pull Request Requirements

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Callbacks command: Update verbiage`
-   [x] The `Because` section summarizes the reason for this PR
-   [x] The `This PR` section has a bullet point list describing the changes in this PR
-   [x] If this PR addresses an open issue, it is linked in the `Issue` section
-   [x] If this PR adds new features or functionality, I have added new tests
-   [x] If applicable, I have ensured all tests related to any command files included in this PR pass, and/or all snapshots are up to date

### CODE REVIEW

**Comment by ManonLef** [comment]
I'd like a few okays from @TheOdinProject/maintainers and @TheOdinProject/moderators before this merge since this PR includes the final state of the remaining text based commands. We can obviously revert some back if we think we shouldn't have deleted one, but a few thumbs up would be comfortable for this final change.

**Review by Mclilzee — APPROVED** [review]
LGTM 🚀

**Review by zachmmeyer — APPROVED** [review]

**Review by 01zulfi — APPROVED** [review]
lgtm 🚀

**Pull request merged**
